prover/src/trace/poly_table.rs

```diff
@@ -74,6 +74,10 @@ impl<E: FieldElement> TracePolyTable<E> {
         for aux_polys in self.aux_trace_polys.iter() {
             result.append(&mut aux_polys.evaluate_columns_at(x));
         }
+
+        if let Some(lagrange_kernel_poly) = &self.lagrange_kernel_poly {
+            result.push(polynom::eval(lagrange_kernel_poly, x));
+        }
```

Comment:
Note: we also need to pass this here so that this column is also available in the OOD frame, needed by the "s" column's transition constraint.